ADAL/src/validation/ADDrsDiscoveryRequest.m

```diff
@@ -30,13 +30,15 @@ @implementation ADDrsDiscoveryRequest
 + (void)requestDrsDiscoveryForDomain:(NSString *)domain
                             adfsType:(AdfsType)type
                              context:(id<MSIDRequestContext>)context
+                     requestMetadata:(NSDictionary *)metadata
```

Comment:
Just curious, do we still need this given that `MSIDRequestContext` already contains `appRequestMetadata`? :)

Reply:
I don't think so, will update :)

Reply:
Addressed.